test/common/policy_utils.go

```diff
@@ -204,6 +205,15 @@ func GetLatestStatusMessage(policyName string, templateIdx int) func() string {
 	}
 }
 
+func DoHistoryUpdatedTest(policyName string, compliance policiesv1.ComplianceState, templateIdx int) {
```

Comment:
Could you please change this so that you are checking the whole history?

Part of the goal of this test is to ensure that not only that the latest status message is correct but that all the states we expected have occurred.

Reply:
Could you please also check the status message in each entry of the history?